Portfolio - "Mobile first approach"

felixDev22 opened the pull request:
**INTRO**
This is a "Mobile first approach" project whose main goal is to  empower the developer with the current trends of software development which emphasis on mobile user experience first.

**In this repo:**

- I created a repository named "Portfolio-Microverse-Project1"
- Cloned the repository in my Visual Code.
- Included the files need for Linters
- Created a branch - "heropage"
- Created the index.html and committed it.
- Created the style.css and committed it

**Output Screenshots**
Here is the look and feel of the project as viewed on Chrome live server.
![portfolio-mobile first](https://user-images.githubusercontent.com/106531695/191636960-4a4c0a16-5e3f-497d-9e54-4e567f6565b7.png)

felixDev22 committed:
Add the linters file

felixDev22 committed:
Add the gitingnore file

felixDev22 committed:
Add hintrc file

felixDev22 committed:
Add stylelintrc file

felixDev22 committed:
Add index.html file

felixDev22 committed:
Add style.css file

felixDev22 committed:
Update index.html file

felixDev22 committed:
Update style.css file

felixDev22 committed:
Add navbar section

felixDev22 committed:
Add the hero-section

felixDev22 committed:
Update index.html

felixDev22 committed:
Update READ.ME.md file

felixDev22 committed:
Update style.css file

franclobo commented on README.md:
```diff
@@ -1,2 +1,42 @@
+
+## Authors
+
+👤 **Author1**
```
- [ ] Please ✔️ replace it with your name because you are the author of your project.

franclobo commented on README.md:
```diff
@@ -1,2 +1,42 @@
+
+Give a ⭐️ ⭐️ if you like this project!.
+
+## Acknowledgments
+
+- Inspired by Microverse in the spirit of empowering developers with knowledge.
```
- [x] Kindly add the LICENSE file to your repository and the LICENSE section to your README file for a professional README file. Please 🙏🏽 make sure the link works as expected.

felixDev22 replied:
I have added as per your feedback. Thanks

![Screenshot 2022-09-21 at 12 28 30](https://user-images.githubusercontent.com/106531695/191702059-700c939f-d9f1-4afb-96bb-bb6b5970c4b6.png)

felixDev22 replied:
I have added the MIT license as per your request. Thanks

![Screenshot 2022-09-22 at 11 44 33](https://user-images.githubusercontent.com/106531695/191702661-24fee4fa-3683-4e53-9620-fc1aefa9457b.png)

franclobo commented on index.html:
```diff
@@ -0,0 +1,61 @@
+        </div>
+        <!-- navigation starts-->
+        <nav>
+          <h2 id="logo">MOZO</h2>
```
- [x] Kindly add an `a` tag to your logo for a better user experience (UX). 👍🏽

franclobo commented on index.html:
```diff
@@ -0,0 +1,61 @@
+        <!-- navigation ends-->
+
+        <!-- hero section starts-->
+        <div class="hero-section">
```
- [x] Please 🙏🏽 use the `section` tag instead of the `div` tag for the best practices of HTML & CSS.

franclobo commented on index.html:
```diff
@@ -0,0 +1,61 @@
+              <a href="#">
+                <img src="images/Vector.png" alt="github" />
+              </a>
+            </div>
```
- [x] Kindly use the `ul` and `li` tags instead of the div tag for the best practices of HTML & CSS. 👍🏽 
- [x] Please 🙏🏽 complete the social icons there are five icons in the Figma template: Tweeter, LinkedIn, Medium, GitHub, and Angellist.
![image](https://user-images.githubusercontent.com/58642949/191640379-adcbd341-126f-4390-804c-4f6165e56cb1.png)

franclobo commented on style.css:
```diff
@@ -0,0 +1,139 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  text-decoration: none;
+  list-style: none;
+  scroll-behavior: smooth;
+}
```
- [x] Kindly add a width of 375px to look similar to the Figma template.

https://user-images.githubusercontent.com/58642949/191640780-369b3031-65b5-4a53-871c-c0710dc916fb.mp4

franclobo commented on index.html:
```diff
@@ -0,0 +1,61 @@
+            <h1>
+              I'm Felix,<br />
+              <span>Welcome</span> to<br />tech world
+            </h1>
```
- [x] It is good to personalize your portfolio 😃,  but at the moment we **must** stick to the design as much as possible (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

franclobo reviewed (changes requested):
Hi @felixDev22, 

# SATATUS: Required Changes ♻️
Good job so far!
You are doing it well 👏🏽 there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Descriptive commits ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [x] kindly add the MIT LICENSE to your repository.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

felixDev22 committed:
Add MIT license on README.md

felixDev22 committed:
Update index.html & styles.css

felixDev22 commented:
- I have added the MIT license
- Updated the Html file and Style accordingly.

Here is the revised look and feel. Thanks @franclobo for the constructive feedback.

![Screenshot 2022-09-22 at 13 33 00](https://user-images.githubusercontent.com/106531695/191725882-6c2fc5b9-409e-463c-bd3a-3615216e7c95.png)

mckent05 commented on index.html:
```diff
@@ -0,0 +1,142 @@
+        </div>
+        <!-- navigation starts-->
+        <nav>
+          <a href="#" id="logo"><h2>MOZO</h2></a>
+          <div class="Menu-bar">
+            <a href="#"
+              ><img src="images/menu-bar.png" class="menu-bar" alt="menu-bar"
+            /></a>
```
[OPTIONAL]- For improved user experience, I will suggest that when users click on your logo, it navigates to the index page. You can add an actual link to your `a` tag 👍 👍

mckent05 commented on index.html:
```diff
@@ -0,0 +1,142 @@
+                    width="18"
+                    height="20"
+                    viewBox="0 0 18 20"
+                    fill="none"
+                    xmlns="http://www.w3.org/2000/svg"
+                  >
+                    <path
+                      d="M17.9852 7.46114C17.9852 6.32124 17.6192 5.25043 16.9205 4.31779C17.1867 3.41969 17.2532 2.27979 17.1867 0.967185C17.1534 0.414508 16.7209 0 16.1885 0C15.8891 0 13.3604 0.0345423 11.8965 1.38169C10.6321 1.1399 9.30128 1.1399 8.00368 1.38169C6.573 0.0345423 4.04435 0 3.71163 0C3.17928 0 2.74675 0.414508 2.71348 0.967185C2.61366 2.27979 2.71348 3.41969 2.97965 4.31779C2.280 ...
+                      fill="#505F79"
+                    />
+                  </svg>
+                </a>
+              </li>
+              <li>
+                <a href="">
+                  <svg
+                    width="15"
+                    height="20"
+                    viewBox="0 0 15 20"
+                    fill="none"
+                    xmlns="http://www.w3.org/2000/svg"
+                  >
+                    <path
+                      d="M6.54751 20C11.2442 20 14.1667 16.5392 14.1667 12.0742C14.1667 10.3908 13.8925 8.87083 11.9817 8.41417C12.4467 7.14083 13.7883 3.4575 13.7883 2.2775C13.7883 1.23833 13.1633 0.3675 12.0492 0.3675C10.2733 0.3675 8.68167 5.51167 8.18334 6.73833C7.7975 5.625 5.99084 0 4.39834 0C3.16084 0 2.58001 0.894167 2.58001 2.01917C2.58001 3.39833 3.94084 6.9725 4.43417 8.3475C4.18417 8. ...
+                      fill="#505F79"
+                    />
+                  </svg>
+                </a>
+              </li>
+            </ul>
+          </div>
+
+          <div class="header-shapes">
+            <img src="images/header-shapes mobile.png" alt="header-shapes" />
+          </div>
+        </section>
+
+        <!-- hero section ends-->
+      </div>
+    </header>
```
**[OPTIONAL]** - To help improve your site's accessibility and SEO benchmarks, the `header` tag should only wrap up your navigation. In your case, it can wrap up both the status bar and navigation. You mage find this [article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) useful in learning more about the `header` tag 😄

felixDev22 replied:
Thank you @mckent05 - I will surely use the feedback to improve my skills.

mckent05 reviewed (approved):
Hi @felixDev22, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions
- I have added some optional changes I feel can help improve your website, Please feel free to implement them. By the way, I like your website, well-done 👏 👏 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
